build.sbt

```diff
@@ -51,6 +51,7 @@ val datastoreV1ProtoClientVersion = "1.6.3"
 // scala-steward:off
 val elasticsearch6Version = "6.8.23"
 val elasticsearch7Version = "7.17.1"
+val elasticsearch8Version = "8.1.2"
```

Comment:
I think we can leave this outside of the `// scala-steward:off` block :) I originally added that because scala-steward kept trying to bump the major versions of ES6 and ES7, but at least until ES9 is released, we can let it update minor versions of ES8.